tests/ltp/patches/chown04.patch

```diff
@@ -0,0 +1,98 @@
+diff --git a/testcases/kernel/syscalls/chown/chown04.c b/testcases/kernel/syscalls/chown/chown04.c
+index 1f3ed412b..95fee2087 100644
+--- a/testcases/kernel/syscalls/chown/chown04.c
++++ b/testcases/kernel/syscalls/chown/chown04.c
+@@ -37,7 +37,8 @@
+  *   6) chown(2) returns -1 and sets errno to ENOENT if the specified file
+  *              does not exists.
+  */
+-
++/* Patch Description: Test cases were failing as there was not sufficient memory to use for test image creation for a loop device for test purpose.
++Solution: Test cases are modified to use /dev/vda the root file system device. In test accessing fault address is commented. This is a known issue in sgx that it causes enclave abort and is being tracked in Issue 169. Also commented testcase which checks mounted filesystem is readonly or not , it has a bug will be tracked under issue 189*/
+ #include <stdio.h>
+ #include <stdlib.h>
+ #include <unistd.h>
+@@ -68,7 +69,7 @@
+ #define TEST_FILE5              "mntpoint"
+
+ static char long_path[PATH_MAX + 2];
+-static const char *device;
++static const char *device = "/dev/vda";
+ static int mount_flag;
+
+ static struct test_case_t {
+@@ -77,12 +78,12 @@ static struct test_case_t {
+ } tc[] = {
+        {TEST_FILE1, EPERM},
+        {TEST_FILE2, EACCES},
+-       {(char *)-1, EFAULT},
++       //{(char *)-1, EFAULT}, will be enabled after github issue 169 is fixed
+        {long_path, ENAMETOOLONG},
+        {"", ENOENT},
+        {TEST_FILE3, ENOTDIR},
+-       {TEST_FILE4, ELOOP},
+-       {TEST_FILE5, EROFS}
++       {TEST_FILE4, ELOOP}
++       //{TEST_FILE5, EROFS} will be enabled after github issue 189 is fixed
+ };
+
+ TCID_DEFINE(chown04);
+@@ -136,7 +137,7 @@ int main(int ac, char **av)
+ static void setup(void)
+ {
+        struct passwd *ltpuser;
+-       const char *fs_type;
++       const char *fs_type = "ext4";
+
+        tst_require_root();
+        tst_sig(FORK, DEF_HANDLER, cleanup);
+@@ -144,24 +145,24 @@ static void setup(void)
+
+        tst_tmpdir();
+
+-       fs_type = tst_dev_fs_type();
+-       device = tst_acquire_device(cleanup);
++       //fs_type = tst_dev_fs_type;
++       //device = tst_acquire_device(cleanup);
+        if (!device)
+                tst_brkm(TCONF, cleanup, "Failed to obtain block device");
+
+-       tst_mkfs(cleanup, device, fs_type, NULL, NULL);
++       //tst_mkfs(cleanup, device, fs_type, NULL, NULL);
```

Comment:
Why commenting out? Why not deleting? Code gets messy with lots of commented out code.